contracts/ForkDelta.sol

```diff
@@ -1,64 +1,12 @@
-pragma solidity ^0.4.20;
+pragma solidity ^0.4.21;
 
-contract SafeMath {
-  function safeMul(uint a, uint b) pure internal returns (uint) {
-    uint c = a * b;
-    assert(a == 0 || c / a == b);
-    return c;
-  }
-
-  function safeSub(uint a, uint b) pure internal returns (uint) {
-    assert(b <= a);
-    return a - b;
-  }
-
-  function safeAdd(uint a, uint b) pure internal returns (uint) {
-    uint c = a + b;
-    assert(c>=a && c>=b);
-    return c;
-  }
-}
-
-contract Token {
-  /// @return total amount of tokens
-  function totalSupply() public constant returns (uint256 supply) {}
-
-  /// @param _owner The address from which the balance will be retrieved
-  /// @return The balance
-  function balanceOf(address _owner) public constant returns (uint256 balance) {}
-
-  /// @notice send `_value` token to `_to` from `msg.sender`
-  /// @param _to The address of the recipient
-  /// @param _value The amount of token to be transferred
-  /// @return Whether the transfer was successful or not
-  function transfer(address _to, uint256 _value) public returns (bool success) {}
+import "./Token.sol";
+import "./SafeMath.sol";
 
-  /// @notice send `_value` token to `_to` from `_from` on the condition it is approved by `_from`
-  /// @param _from The address of the sender
-  /// @param _to The address of the recipient
-  /// @param _value The amount of token to be transferred
-  /// @return Whether the transfer was successful or not
-  function transferFrom(address _from, address _to, uint256 _value) public returns (bool success) {}
+contract ForkDelta {
+  
+  using SafeMath for uint;
 
-  /// @notice `msg.sender` approves `_addr` to spend `_value` tokens
```

Comment:
Why was this all removed?

Reply:
It went into its own files Token.sol and SafeMath.sol.